src/averageAges.js

```diff
@@ -20,6 +20,20 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const findMen = arguments.length < 2
+    ? people.filter(listOfMen =>
+      listOfMen.sex === 'm')
+
+    : people.filter(listOfMen =>
+      Math.ceil(listOfMen.died / 100) === century && listOfMen.sex === 'm');
```

Comment:
you do the same thing in these 2 filter, by sorting out men. better filter out men first, and then, if needed filter them out by century
```suggestion
    ? people.filter(listOfMen =>
      listOfMen.sex === 'm')
    : people.filter(listOfMen =>
      Math.ceil(listOfMen.died / 100) === century && listOfMen.sex === 'm');
```